client/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <link href="/src/favicon.svg" rel="icon" type="image/svg+xml" />
```

Comment:
Need to change favicon and the title

Reply:
Use Logux favicon

Reply:
You can use this URL it is permament https://logux.io/favicon.ico